npm/react/examples/nextjs/package.json

```diff
@@ -19,7 +19,10 @@
     "@zeit/next-sass": "^1.0.1",
     "check-code-coverage": "1.9.2",
     "cypress-circleci-reporter": "0.2.0",
-    "next": "10.1.2"
+    "next": "10.1.2",
+    "@cypress/webpack-dev-server": "^1.1.3"
```

Comment:
Missing a comma. so npm install is failing on CI. Can you add comma?

Reply:
Ah damn, of course

Reply:
Fixed @lmiller1990 💪